cristin-import/src/main/java/no/unit/nva/cristin/lambda/FakeHttpClient.java

```diff
@@ -0,0 +1,100 @@
+package no.unit.nva.cristin.lambda;
```

Comment:
This class exists twice. Once in the testing library, and once here only for the cristin service. They give a different response, but the main reason for the replication is that I do not want to include all testing libraries in the main code.